src/App.tsx

```diff
@@ -15,71 +15,94 @@ export const goods = [
   'Garlic',
 ];
 
-export const App: React.FC = () => (
-  <main className="section container">
-    <h1 className="title">No goods selected</h1>
+interface State {
+  selectedGood: string | null;
+  isSelected: boolean;
```

Comment:
The 'isSelected' state is not necessary. You can determine if a good is selected by checking if 'selectedGood' is not null or an empty string.